src/zenml/zen_stores/schemas/user_management_schemas.py

```diff
@@ -35,13 +36,31 @@
 class TeamAssignmentSchema(SQLModel, table=True):
     """SQL Model for team assignments."""
 
-    user_id: UUID = Field(primary_key=True, foreign_key="userschema.id")
-    team_id: UUID = Field(primary_key=True, foreign_key="teamschema.id")
+    __tablename__ = "team_assignment"
+
+    user_id: UUID = build_foreign_key_field(
+        source=__tablename__,
+        target="user",  # TODO: how to reference `UserSchema.__tablename__`?
```

Comment:
Are these todo's here still relevant?

Reply:
If there's a way to do it I would like to reference the class instead of hardcoding the table name, but it seems like a cyclic import issue

Reply:
I fixed all the TODOs that were fixable. The only ones left are in tables that are used as `link_model` somewhere. In these cases, the class of the foreign key tables cannot be referenced since those classes need to reference the link model already.